packages/terra-functional-testing/src/services/wdio-visual-regression-service/VisualRegressionLauncher.js

```diff
@@ -104,14 +104,16 @@ class VisualRegressionLauncher {
 
       const screenshotContext = {
         browserInfo: this.context.browserInfo,
+        desiredCapabilities: this.context.desiredCapabilities,
         suite: this.currentSuite,
         test: this.currentTest,
         meta: {
           currentFormFactor,
         },
+        options,
```

Comment:
these options are passed to the screenshot command on line 116, do they need to be added to the screenshot context as well?

Reply:
The reason I added options to the context is to get `options.name` to use for the screenshot name. It currently creates the screenshot name using the context in BaseCompare: https://github.com/cerner/terra-toolkit/pull/545/files#diff-dea71618479daf3d0e21b24c6f7201de0dc18481e558caf0c3aee745a434af06R71

Reply:
Gottchya that makes sense. Could we just pass the name? no preference there
Can we add `name` to the docs?

Reply:
Added `name` to docs. https://github.com/cerner/terra-toolkit/pull/545/commits/9933873194e2d7e6d395853f313cdd4cf89d1212